app/League-Entries/page.tsx

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Gridiron Survivor.
```

Comment:
This file location will change. I'm getting feedback on what it should be.

Reply:
As mentioned in my PR description, the page is only there to see the rendered components and I was going to delete it upon approval as I know it was a separate ticket to create the page.

Reply:
Leave it for now. We'll have the FEE team move it.

Reply:
Got it!